core/src/main/java/nl/nn/adapterframework/core/Adapter.java

```diff
@@ -85,6 +85,9 @@
 public class Adapter implements IAdapter, NamedBean {
 	private Logger log = LogUtil.getLogger(this);
 	protected Logger msgLog = LogUtil.getLogger("MSG");
+	
+	public static final String PROCESS_STATE_OK = "OK";
+	public static final String PROCESS_STATE_ERROR = "ERROR";
```

Comment:
PROCESS_STATE_ERROR or "ERROR" is a duplicate from errorState: https://github.com/ibissource/iaf/blob/master/core/src/main/java/nl/nn/adapterframework/core/Adapter.java#L134

I would use an enum instead:

```
diff --git a/core/src/main/java/nl/nn/adapterframework/core/Adapter.java b/core/src/main/java/nl/nn/adapterframework/core/Adapter.java
index bbb33dd04..4d84c3f78 100644
--- a/core/src/main/java/nl/nn/adapterframework/core/Adapter.java
+++ b/core/src/main/java/nl/nn/adapterframework/core/Adapter.java
@@ -86,9 +86,6 @@ public class Adapter implements IAdapter, NamedBean {
        private Logger log = LogUtil.getLogger(this);
        protected Logger msgLog = LogUtil.getLogger("MSG");
        
-       public static final String PROCESS_STATE_OK = "OK";
-       public static final String PROCESS_STATE_ERROR = "ERROR";
-
        private String name;
        private Configuration configuration;
        private String targetDesignDocument;
@@ -96,7 +93,12 @@ public class Adapter implements IAdapter, NamedBean {
 
        private ArrayList<IReceiver> receivers = new ArrayList<IReceiver>();
        private long lastMessageDate = 0;
-       private String lastMessageProcessingState; //"OK" or "ERROR"
+
+       public enum PROCESS_STATE {
+               OK, ERROR
+       }
+
+       private String lastMessageProcessingState;
        private PipeLine pipeline;
 
        private Map<String, SenderLastExitState> sendersLastExitState = new HashMap<String, SenderLastExitState>();
@@ -258,9 +260,9 @@ public class Adapter implements IAdapter, NamedBean {
                        numOfMessagesProcessed.increase();
                        statsMessageProcessingDuration.addValue(duration);
                        if (processingSuccess) {
-                               lastMessageProcessingState = PROCESS_STATE_OK;
+                               lastMessageProcessingState = PROCESS_STATE.OK.toString();
                        } else {
-                               lastMessageProcessingState = PROCESS_STATE_ERROR;
+                               lastMessageProcessingState = PROCESS_STATE.ERROR.toString();
                        }
                        notifyAll();
                }
diff --git a/core/src/main/java/nl/nn/adapterframework/webcontrol/pipes/ShowConfigurationStatus.java b/core/src/main/java/nl/nn/adapterframework/webcontrol/pipes/ShowConfigurationStatus.java
index 24ecc3e02..a050b3767 100644
--- a/core/src/main/java/nl/nn/adapterframework/webcontrol/pipes/ShowConfigurationStatus.java
+++ b/core/src/main/java/nl/nn/adapterframework/webcontrol/pipes/ShowConfigurationStatus.java
@@ -470,9 +470,9 @@ public class ShowConfigurationStatus extends ConfigurationBase {
                } else {
                        adapterXML.addAttribute("lastMsgProcessState", "-");
                }
-               if (Adapter.PROCESS_STATE_OK.equals(lastMsgProcessingState)) {
+               if (Adapter.PROCESS_STATE.OK.equals(lastMsgProcessingState)) {
                        showConfigurationStatusManager.countLastMsgProcessOk++;
-               } else if (Adapter.PROCESS_STATE_ERROR.equals(lastMsgProcessingState)) {
+               } else if (Adapter.PROCESS_STATE.ERROR.equals(lastMsgProcessingState)) {
                        showConfigurationStatusManager.countLastMsgProcessError++;
                        if (!showConfigurationStatusAdapterManager.stateAlert) {
                                showConfigurationStatusAdapterManager.stateAlert = true;
```